engine/src/main/java/org/terasology/rendering/dag/SkyBandsNode.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag;
+
+import org.terasology.config.Config;
+import org.terasology.config.RenderingConfig;
+import org.terasology.config.RenderingDebugConfig;
+import org.terasology.monitoring.PerformanceMonitor;
+import org.terasology.registry.In;
+import org.terasology.rendering.assets.material.Material;
+import org.terasology.rendering.cameras.Camera;
+import org.terasology.rendering.opengl.FBO;
+import org.terasology.rendering.opengl.FrameBuffersManager;
+import org.terasology.rendering.world.WorldRenderer;
+
+import static org.lwjgl.opengl.GL11.GL_COLOR_BUFFER_BIT;
+import static org.lwjgl.opengl.GL11.GL_DEPTH_BUFFER_BIT;
+import static org.lwjgl.opengl.GL11.glClear;
+import static org.terasology.rendering.opengl.OpenGLUtils.*;
+
+
+/**
+ * TODO: Diagram of this node
+ * TODO: Separate this node into multiple SkyBandNode's
+ */
+public class SkyBandsNode implements Node {
+
+    @In
+    private Config config;
+
+    @In
+    private WorldRenderer worldRenderer;
+
+    @In
+    private FrameBuffersManager frameBuffersManager;
+
+    private RenderingConfig renderingConfig;
+    private Material blurShader;
+    private FBO sceneOpaque;
+    private FBO sceneSkyBand0;
+    private RenderingDebugConfig renderingDebugConfig;
+    private Camera playerCamera;
+
+    @Override
+    public void initialise() {
+        renderingConfig = config.getRendering();
+        renderingDebugConfig = renderingConfig.getDebug();
+        blurShader = worldRenderer.getMaterial("engine:prog.blur");
+        playerCamera = worldRenderer.getActiveCamera();
+    }
+
+    @Override
+    public void process() {
+        enableWireframeIf(renderingDebugConfig.isWireframe());
+
+        sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
+
+        setRenderBufferMask(sceneOpaque, true, true, true);
+        if (renderingConfig.isInscattering()) {
+            sceneSkyBand0 = frameBuffersManager.getFBO("sceneSkyBand0");
+            FBO sceneSkyBand1 = frameBuffersManager.getFBO("sceneSkyBand1");
+
+            generateSkyBand(sceneSkyBand0);
+            generateSkyBand(sceneSkyBand1);
+        }
+
+        sceneOpaque.bind();
+
+        playerCamera.lookThrough();
+        endRenderSkyActivity();
+    }
+
+    private void endRenderSkyActivity() {
+        PerformanceMonitor.endActivity();
```

Comment:
You mean like:
``` java
PerformanceMonitor.startActivity("rendering/backdrop") // Executed inside Backdrop
...
PerformanceMonitor.startActivity("rendering/backdrop/skybands") // Executed inside SkyBand node
... 
PerformanceMonitor.endActivity("rendering/backdrop/skybands") // Executed inside SkyBand 

PerformanceMonitor.endActivity("rendering/backdrop") // Executed not inside SkyBand, but inside Backdrop
```

But the activity backdrop (`rendering/backdrop`) in total is going to include measurements from `rendering/backdrop/skybands` right?